model/Security/Properties/modified.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# modified
```

Comment:
I've changed this but I keep referring back to the notion of the node's  immutability, this may be useless as it may have to be changed with an `AMENDS`. Leaving it for now in case it models an existing data entry  schema.